docs/common/releases/release-notes.md

```diff
@@ -66,7 +66,21 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 </dependencies>
 ```
 
-## 0.13.0 Release Notes
+## 0.13.1 Release Notes
+
+**Important bug fixes:**
+
+* **Spark**
+  * Fix predicate pushdown (filter-based and merge conditions) for row-level operations in Spark 3.2. [[\#4023](https://github.com/apache/iceberg/pull/4023)]
+  * Fix an issue introduced in [#3722](https://github.com/apache/iceberg/pull/3722) and in the 0.13.0 release which lead to not being able to
```

Comment:
No need to mention PR or release that brings in the issue, better to describe the symptom so people can know if the fix is related to their usage.

Reply:
Updated so it's more concise